payment-gateways-by-user-roles-for-woocommerce.php

```diff
@@ -118,6 +118,11 @@ public function __construct() {
 		 * @version 1.2.1
 		 * @since   1.0.0
 		 */
+		/**
+ 		* Core instance for frontend and admin.
+ 		* @var object $core
+ 		*/
+		 protected $core;

```

Comment:
this section too needs corrections. This section must move above the function comments